selfdrive/car/honda/hondacan.py

```diff
@@ -76,6 +136,7 @@ def create_ui_commands(packer, pcm_speed, hud, car_fingerprint, is_metric, idx,
       "FCM_PROBLEM": stock_hud["FCM_PROBLEM"],
       "ICONS": stock_hud["ICONS"],
     }
+  if radar_disabled:
```

Comment:
maybe pass in CarParams instead? this name is confusing, and CarParams has the fingerprint

Reply:
What is confusing about `radar_disabled`? It literally means the radar has been disabled and is not outputting any messages.

Reply:
also, the fingerprint by itself doesn't provide enough info (we don't want to output `ACC_HUD` if the radar is also outputting it)

Reply:
oh I see now, this isn't whether the radar is disabled, it is whether openpilot is handling longitudinal control, I will fix the name

Reply:
fixed